utils/sl/components/__init__.py

```diff
@@ -287,7 +287,8 @@ def define_components(reqs):
                 commands=['git clean -dxf ',
                           './autogen.sh',
                           './configure --prefix=$ARGOBOTS_PREFIX CC=gcc'
-                          ' --enable-valgrind',
+                          ' --enable-valgrind'
+			  ' --with-libunwind=/usr/lib64 --enable-stack-unwind',
```

Comment:
Does `--with-libunwind=` really require a path when it's a system standard path?  `configure` should usually be able to check standard system paths automatically. If it cannot, that sounds like a bug in argobots' `configure` and should be reported upstream.

Reply:
Hmm, you are right Brian, I forgot to remove this change prior to push patch, BTW the path is even wrong!, sorry about this and thx to notice it !